src/printer.rs

```diff
@@ -39,6 +64,8 @@ impl Printer {
                 }
             }
         }
+        self.line_lengths = SVec::new();
```

Comment:
We could add a `clear()`function to SVec later

Reply:
yup